src/js/Controllers/UIController.js

```diff
@@ -213,6 +262,13 @@ class UIController {
                     this.listener.send(RpcFactory.AlertAbortedResponse(app.alert.msgID))
                     store.dispatch(closeAlert(app.alert.msgID, rpc.params.appID))
                     return true
+                } else if (rpc.params.functionID === 64 && app.alert.showAlert
```

Comment:
Please use a constant or add a note stating that functionID 64 refers to `SubtleAlertID`

Reply:
Marking as resolved for this PR. Please refer to https://github.com/smartdevicelink/generic_hmi/pull/287#discussion_r469189664